src/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
+find_package(Eigen3 CONFIG REQUIRED)
+
 add_library(WaveGrid WaveGrid.cpp Enviroment.cpp Grid.cpp ProfileBuffer.cpp Spectrum.cpp ${Environment_RCS})
+target_link_libraries(WaveGrid Eigen3::Eigen)
```

Comment:
Eigen3 does not have any libraries, it's header only?!

Reply:
That's required to setup the correct include paths for the project. Maybe that's a MSVC thing but I don't think it hurts anyway.

Reply:
Done